docs/howto/python/packages.md

```diff
@@ -0,0 +1,36 @@
+# Use additional Python packages
+
+## Installing packages at runtime
+
+JupyterLite provides an additional `piplite` layer on top of [micropip] to install
+packages from Python notebooks.
+
+New packages can be downloaded and installed at runtime. For example to install
+`snowballstemmer`.
+
+```py
+import piplite
+await piplite.install("snowballstemmer")
+```
+
+```{note}
+Some Python packages require a frontend extension to also be installed.
+
+See [](../../howto/configure/simple_extensions.md) for more information.
+
+See also how to [ship additional wheels](./wheels.md) at build time.
+```
+
+[micropip]: https://pyodide.org/en/latest/usage/loading-packages.html?#micropip
+
+## Bundling additional packages by default
```

Comment:
@martinRenou once it is ready for prime time, we can update the documentation to mention how to do this with the xeus python kernel.

Thinking this could go in a `0.2.0` release, also after we have moved the Pyolite kernel to a separate repo (https://github.com/jupyterlite/jupyterlite/issues/386)

Reply:
> I don't fully know what that looks like, but it seems like having both python kernels (and their downstream wrappers) being able to rely on some common configuration source would make a lot of sense.

Yeah I'm not sure whether it will be possible to unify the configuration for both kernels here. It's also not clear if or when xeus python and pyodide might converge at some point if they start sharing more common parts.